content/applications/marketing/marketing_automation/campaign_templates/double_optin.rst

```diff
@@ -0,0 +1,153 @@
+=============
+Double opt-in
+=============
+
+A double opt-in, or also referred to as a confirmed opt-in, may be required in some countries for
+marketing communications due to anti-SPAM laws. Confirming consent also has several benefits, as it
+validates email addresses, avoids spam/robo subscribers, helps keep mailing lists clean, and only
+includes engaged contacts in the mailing list.
+
+The *Marketing Automation* app's *Double Opt-in* campaign template automatically configures a new
+campaign to send an email to new recipients to confirm their consent.
+
+When the *Double Opt-in* campaign template is used, a new mailing list titled *Confirmed contacts*
+is created in the *Email Marketing* app, and any new mailing list contacts that are added to the
+default *Newsletter* mailing list are sent a confirmation email to double opt-in. The contacts that
+click on a link in the email are automatically added to the *Confirmed contacts* mailing list.
+
+.. important::
+   When using the *Double Opt-in* campaign template, only the contacts in the *Confirmed contacts
+   mailing* list are considered to have confirmed their consent.
+
+.. _marketing_automation/template/using-double-optin:
+
+Use the Double Opt-in campaign template
+=======================================
+
+Open the :menuselection:`Marketing Automation` app, and select the :guilabel:`Double Opt-in`
+campaign template to create a new campaign for confirming consent.
+
+.. tip::
+   The campaign templates do not display by default when there are existing *Marketing Automation*
+   campaigns. To display the campaign templates, type the name of a campaign that does not exist in
+   the database into the :guilabel:`Search...` bar, then press :kbd:`Enter`.
+
+   For example, searching for `empty` displays the campaign template cards again, as long as there
+   is not a campaign with the name "empty" in the database.
+
+Campaign configuration
+----------------------
+
+Upon creation of the campaign, the campaign form loads with a new preconfigured campaign.
+
+The :guilabel:`Target` and :guilabel:`Filter` configuration of the campaign is as follows:
+
+- :guilabel:`Name`: `Double Opt-in`
+- :guilabel:`Responsible`: The user who created the campaign.
```

Comment:
I'm actually not seeing a 'Responsible' field on the 17.0 runbot I'm using for this review --- but that could be a RunBot issue haha -- worth a double-check, either way, though...
![Screenshot 2024-05-29 110228](https://github.com/odoo/documentation/assets/73958186/47b75e0d-5378-4dba-a4a3-fa30a1e101d9)

Reply:
The Responsible field only shows with developer mode active, I will make a note :)